app/models/metric/long_term_averages.rb

```diff
@@ -62,7 +62,8 @@ def self.get_averages_over_time_period(obj, options = {})
       Metric::Aggregation::Process.average(c, nil, results[:avg], counts)
 
       begin
-        results[:dev][c]  = vals[c].stddev
+        results[:dev][c] = vals[c].length == 1 ? 0 : vals[c].stddev
```

Comment:
I think we should probably fix this in more_core_extensions instead.  Please comment on https://github.com/ManageIQ/more_core_extensions/pull/28